core/controllers/voice_artist.py

```diff
@@ -114,7 +116,7 @@ def post(self, exploration_id):
             '%s/%s' % (self._FILENAME_PREFIX, filename),
             raw_audio_file, mimetype=mimetype)
 
-        self.render_json({'filename': filename})
+        self.render_json({'filename': filename, 'duration': duration})
```

Comment:
make sure to add this duration in handler test.

Reply:
So I am going guess that I have create a test scenario.

Reply:
When you refer to the handler test is it the one in resources_test.py?

Reply:
Done via the resource_test.py to upload and confirm the duration_sec attribute is included and is the correct value.

Reply:
Done.